api/illumio/cloud/k8sclustersync/v1/k8s_info.proto

```diff
@@ -87,11 +97,11 @@ message KubernetesResourceMutation {
   // The documented types
   oneof mutation {
```

Comment:
Do we send the `kind_specific` data for every kind of mutation, or only for create/update?
I think we can omit that for `delete_resource` mutations, as a small optimization.
Clarify that in the comments.

This is really a nit. I'm OK with ignoring this.